### ISSUE TITLE
Not possible to update an OC after removing an entreprise fee 

### ISSUE THREAD

**Issue by myriamboure** [openfoodfoundation/openfoodnetwork#1184]
Before:
![screenshot from 2016-10-11 15-07-22](https://cloud.githubusercontent.com/assets/8135376/19271399/dce4b312-8fc4-11e6-8fd3-571ca21c6231.png)

After: the red "update" button doesn't show up
![screenshot from 2016-10-11 15-07-29](https://cloud.githubusercontent.com/assets/8135376/19271401/de5c89e0-8fc4-11e6-9e4f-b29781c557b1.png)

**Comment by CynthiaReynolds** [comment]
if you make a change elsewhere in any text box, it will give you the update option and allow the change

**Comment by myriamboure** [comment]
Yes I saw the work around, not for the user experience not intuitive at all!
The problem also happens when you delete coordinator fee, so I suspect any fee.

**Comment by lin-d-hop** [comment]
Closing at this is a duplicate of #1165
